src/pages/understand-stacks/mining.md

```diff
@@ -17,7 +17,7 @@ A new Stacks block may be mined once per Bitcoin block. To be considered for min
 
 ## Coinbase Rewards
 
-Miners receive coinbase rewards for blocks they mine. These rewards are shared (proportional to the amount committed to) with an user burn support operations that support the block.
+Miners receive coinbase rewards for blocks they mine.
```

Comment:
Just FYI we'll have to update this for microblocks: https://github.com/stacksgov/sips/blob/main/sips/sip-001/sip-001-burn-election.md#building-off-the-latest-block